library/cartridge_connect_to_membership.py

```diff
@@ -16,8 +16,41 @@
 
 def probe_server(control_console, uri):
     return control_console.eval_res_err('''
-        local advertise_uri = ...
-        return require('cartridge').admin_probe_server(advertise_uri)
+        local advertise_uris = ...
+
+        local cartridge = require('cartridge')
+        local fiber = require('fiber')
+
+        local count = 0
+        local channel = fiber.channel()
+
+        for _, advertise_uri in ipairs(advertise_uris) do
+            count = count + 1
+
+            fiber.create(function()
+                local _, err = cartridge.admin_probe_server(advertise_uri)
+                channel:put({advertise_uri, err})
```

Comment:
Why not a table?

`{advertise_uri = advertise_uri, err = err}`

I don't like using `res[2]` below, like `local err = res[2].err or res[2]`.